symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java

```diff
@@ -100,6 +100,42 @@ protected SymphonyBdk(BdkConfig config, ApiClientFactory apiClientFactory)
     this.activityRegistry = this.datafeedLoop != null ? new ActivityRegistry(this.botInfo, this.datafeedLoop) : null;
   }
 
+  protected SymphonyBdk(BdkConfig config, ApiClientFactory apiClientFactory, AuthenticatorFactory authenticatorFactory)
```

Comment:
should this constructor be public or should I create another public one initialising the ApiClientFactory like is done for the other one. 
```
public SymphonyBdk(BdkConfig config, AuthenticatorFactory authenticatorFactory) throws AuthInitializationException, AuthUnauthorizedException {
    this(config, new ApiClientFactory(config), authenticatorFactory);
  }
```